code/game/gamemodes/shadowling/shadowling.dm

```diff
@@ -164,6 +164,10 @@ Made by Xhuis
 		to_chat(new_thrall_mind.current, "<span class='shadowling'>You may communicate with your allies by speaking in the Shadowling Hivemind (:8).</span>")
 		if(jobban_isbanned(new_thrall_mind.current, ROLE_SHADOWLING) || jobban_isbanned(new_thrall_mind.current, ROLE_SYNDICATE))
 			replace_jobbanned_player(new_thrall_mind.current, ROLE_SHADOWLING)
+		if(!victory_warning_announced && (length(shadowling_thralls) >= (required_thralls - 3)))//are the slings very close to winning?
```

Comment:
I suggest a larger gap than 3. Perhaps 6. With 3, there's no time to react.

Reply:
Since required thralls scales with players I think it makes sense to make the victory warning threshold a portion of the required thralls. Something like 66-80%, I'd say.

Reply:
66% rounded would be the best imo, by 66% they are usually pretty close to snowballing to victory.